file_upload_handler/src/server.js

```diff
@@ -0,0 +1,59 @@
+import express from "express";
+import multer from "multer";
+
+const PORT = process.env.PORT || 8000
+const app = express();
+
+const storage = multer.diskStorage({
+  destination: function (req, file, cb) {
+    cb(null, "./uploads");
+  },
+  filename: function (req, file, cb) {
+    const uniquepreffix = Date.now() + "-" + Math.round(Math.random() * 1e9);
```

Comment:
Please be consistent with your naming convension

In this case `uniquepreffix` should be `uniquePreffix`